tests/features/java/memory.feature

```diff
@@ -13,6 +13,27 @@ Feature: OPENJDK-559 JVM Memory tests
     | JAVA_MAX_MEM_RATIO | 90.0   |
     Then container log should contain -XX:MaxRAMPercentage=90.0
 
+  @ubi8
+  Scenario: Ensure JAVA_MAX_MEM_RATIO accepts Integers
+    Given container is started with env
+    | variable           | value  |
+    | JAVA_MAX_MEM_RATIO | 90     |
+    Then container log should contain -XX:MaxRAMPercentage=90.0
+
+  @ubi8
+  Scenario: Ensure JAVA_INITIAL_MEM_RATIO accepts Integers
+    Given container is started with env
+    | variable           | value  |
+    | JAVA_MAX_MEM_RATIO | 10     |
+    Then container log should contain -XX:MaxRAMPercentage=10.0
```

Comment:
`JAVA_INITIAL_MEM_RATIO` instead of `JAVA_MAX_MEM_RATIO` and `-XX:InitialRAMPercentage=10.0` instead of `-XX:MaxRAMPercentage=10.0`